docs/apis-clients/cli-client/cli-get-started.md

```diff
@@ -65,7 +65,7 @@ Brokers:
 
 ## Advanced process
 
-Use [this process model](./assets/gettingstarted_quickstart_advanced.bpmn) for the tutorial.
```

Comment:
I don't actually think I needed to update this...both the left and right version are treated the same. I probably did this as an experiment early on, while learning which relative paths I needed to worry about. 

I can undo this change if people would like; it is a harmless change, though.